app/appsettings.json

```diff
@@ -2,8 +2,8 @@
   "WordPress": {
     "dbhost": "localhost",
     "dbpassword": "password",
-    "dbuser": "root",
```

Comment:
this breaks the `readme` instructions:

1. Start mysql, at localhost:3306, user **root**, password password, created database **wordpress**

Reply:
oops, that was an accident. my bad :-)